### DISCUSSION TITLE
Is it possible to have more than 1 Custom TARDIS interior per server?

### DISCUSSION THREAD

**Discussion by TuckerAndrews3** [eccentricdevotion/TARDIS#688] [General]
I am seeing on this page that it has to be named custom and the implication is that there can only be 1

https://eccentricdevotion.github.io/TARDIS/schematics.html

Is that correct? I am trying to make a bunch of custom consoles

**Comment by TuckerAndrews3** [comment]
Nevermind, figured it out!

**Comment by eccentricdevotion** [comment]
For others who may come across this:

Multiple [custom consoles](https://eccentricdevotion.github.io/TARDIS/custom-consoles.html) are possible.
* Create the schematics using the instructions on the [schematics page](https://eccentricdevotion.github.io/TARDIS/schematics.html)
* Add entries to _custom_consoles.yml_ and _artron.yml_
* Restart the server to load the new consoles

**Reply by eccentricdevotion** [reply]
I'll update the docs to make them a bit clearer...

**Comment by eccentricdevotion** [comment]
It's all explained in more detail on the Custom Consoles page - [https://eccentricdevotion.github.io/TARDIS/custom-consoles.html](https://eccentricdevotion.github.io/TARDIS/custom-consoles.html)
